### DISCUSSION TITLE
Our big idea

### DISCUSSION THREAD

**Discussion by lulen11** [codelikeagirlau/accessibility-resources#6] [Polls]
Hey CLG Community!! We're so excited to be co-creating a project for #hacktoberfest with you! As we've shared, we would love to theme it on #accessibility! If you have any ideas, please add them to the poll and/or vote on your favourite!

**Comment by sarahesbie** [comment]
Heya team! I'm excited...

I was just checking out the [a11y project](https://www.a11yproject.com/) shared by Lauren in another discussion. 
We don't want to re-invent something that someone else has done, so I wonder if we broaden our horizon beyond WCAG-specific accessibility. 

I keep coming back to the thought of "how do we make the web better, more inclusive and more accessible for everyone?"
WCAG is obviously a big part of this.
But also... how do we make sure the web is inclusive of intersectional groups? How do we ensure the language we use on the web is accessible and inclusive? How can we be inclusive when posting to social media sites, email, Slack, chat etc? What about IOT devices and non-screen interfaces? How can we improve user experiences? How can we educate people about dark patterns? How can we make things easier for people who aren't tech savvy? What about the future and things that haven't been invented yet?

This is a bit of a ramble and big picture thinking... but I'm thinking that our project could include things like:

- a manifesto on "making the web better for everyone"
- collections of resources and tools for web accessibility and web inclusivity
- outbound links to relevant articles 
- checklists

**Comment by wendy-ha18** [comment]
I'm also know that many organizations have shared web accessibility checklists and standards. However, I'm still keen to the idea of creating a "collections" or "checklists" or "cheat sheet" repo (I love cheat sheet :D)

02 #Hacktoberfest repos that currently I find intriguing and that share a similar concept are [App-Ideas](https://github.com/florinpop17/app-ideas) and [Awesome Datascience](https://github.com/academic/awesome-datascience).

Hence, I have some ideas for our project as below, hopefully they can be useful :
- A HTML/CSS/iOS/Figma Accessibility cheat sheet.

- A collection of AI assistive tools and applications. These could encompass a wide range of technologies, such as translation between written and spoken languages to sign languages, image recognition, facial recognition, lip reading, etc. (Reference this repo: [Awesome Assistive Tech](https://github.com/openassistive/awesome-assistivetech))

- Taking inspiration from the [Awesome Accessibility](https://github.com/brunopulis/awesome-a11y/tree/main#awesome-accessibility) repo and what @sarahesbie shared above, we could incorporate sections dedicated to Tools for Accessibility checking, Tech Companies & Organizations providing Accessibility services, Podcasts, Articles, or People to Follow in the field of web accessibility.

- Additionally, we could make a list of project ideas suitable for accessibility-theme hackathons or list of hackathon aimed for accessibility (not sure in Au but I can find a few in other countries) for anyone who interested.

**Reply by sarahesbie** [reply]
Love these ideas @wendy-ha18 ! Feel free to start working on your own feature branches for any of these and sending a pull request when you're ready!

Feel free to add headings, sub headings etc into the readme  as it currently is, to create categories for podcasts, articles, people etc if you have some to list :)

**Comment by laurenchamps** [comment]
I like the idea of bringing together existing resources so the info is easy to find, and also agree with what Sarah has shared about inclusive language, making the web more accessible for non-tech savvy etc etc. I'm not sure what the answers are to many of those questions but they're good things to be thinking about!! 
Because I'm a newbie, and haven't been working in the field, I'm not so familiar with where the gaps are in terms of checklists/cheat sheets that may already exist vs what is needed but very happy to be guided on that and contribute to something that fills those gaps.

**Comment by sarahesbie** [comment]
Should we start thinking about this as a website?

- Static HTML site, or
- Front end library or framework such as [Gatsby](https://www.gatsbyjs.com/), [Jekyll](https://jekyllrb.com/), [Preact](https://preactjs.com/), [Vue](https://vuejs.org/)? 

**Pros and cons**
- static HTML site will be super easy to get up and running, but is not scalable. Short term solution
- Jekyll seems to be [really easy to use with GitHub pages](https://jekyllrb.com/). It's a Ruby framework so might be a bit of a learning curve for some of our community?
- Looks like we can also [deploy Gatsby with GitHub pages](https://www.gatsbyjs.com/docs/how-to/previews-deploys-hosting/how-gatsby-works-with-github-pages/). Gatsby is based on React which is based on JavaScript so learning curve won't be so high.
- I haven't worked with Vue or Preact but I'm always curious about trying out new frameworks.

Would love to hear thoughts and other suggestions!

**Reply by laurenchamps** [reply]
I think a website is a good idea. The scalable solution using a static site generator seems like the way to go but as to which one... I've heard of Gatsby and it seems popular, although from what I understand it may be more complex compared to Jekyll? But if there are a number of people familiar with React then perhaps it would be a good option. I don't know React or Ruby so will be learning curve for me whichever!

Hopefully others may have experience working with them and have some more input.

**Reply by wendy-ha18** [reply]
+1 vote for website as well

**Reply by jazzicoda** [reply]
+1 for website too
As for static/framework... maybe html to start with? as quick and easy (already 9 October) - then could be developed to scalable project next hacktoberfest?

**Reply by sarahesbie** [reply]
Nice, I feel like we're all on the same page. And I agree, let's get up and running with a simple static HTML site and then we can look at a framework later if we want to expand it. 

I'll get started with a super basic HTML file and we can go from there!

**Comment by lulen11** [comment]
Nice one team! I added a bit of a look and feel for our new HTML & CSS website today. And I can see a lot of chat around cheatsheets and checklists here—love that! I love the big picture thinking about how we can make it _all_ more accessible too. 

There would be many resources, cheatsheets etc out there already but whose to say ours won't be another great one to add to the mix! Don't let that deter you ;) We can feature them in our creation if they're really great!  

With us nearing mid-October, let's keep the plan simple and build as we go! Feel free to pull the repo down and add any updates directly to the project as you like 👏 👏
